### PR TITLE
refactor(test_utils): move to common folder

### PR DESCRIPTION
**Pull request by zulfaqar-azmi-t4**
## Description

Other component might benefit from using `test_utils` function, therefore this PR aims to move `test_utils` to `common` folder.
Also, added autoware prefix and namespace

## Related links

None

## Tests performed

Pass CI

## Notes for reviewers

None

## Interface changes

None

### ROS Topic Changes

None

### ROS Parameter Changes

None

## Effects on system behavior

None

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: planning/obstacle_velocity_limiter/test/test_obstacle_velocity_limiter_node_interface.cpp
Comment (maxime-clem) [inline planning/obstacle_velocity_limiter/test/test_obstacle_velocity_limiter_node_interface.cpp:1]: This file should be deleted (this node was moved to be a module of the `motion_velocity_planner`).

Comment (zulfaqar-azmi-t4) [reply planning/obstacle_velocity_limiter/test/test_obstacle_velocity_limiter_node_interface.cpp:1]: Fix in https://github.com/autowarefoundation/autoware.universe/pull/7158/commits/9e022d360bd31509d7680e39aa0e0e9adb0c6a2c

File: planning/autoware_planning_test_manager/README.md
[the remark this answers is not in the document]
Comment (zulfaqar-azmi-t4) [reply planning/autoware_planning_test_manager/README.md:1]: Thank you for the catch!
It originally has `planning_test_utils::` in it, when during i refactor, I just replace the name as is.
It is now fixed in https://github.com/autowarefoundation/autoware.universe/pull/7158/commits/ca1e5b38ab56f0b7864f1632a5b7ba9246a2b496

**Review by maxime-clem — APPROVED** [review]

**Review by kyoichi-sugahara — APPROVED** [review]
LGTM! Thanks

**Review by takayuki5168 — APPROVED** [review]

**Review by rej55 — APPROVED** [review]

**Pull request merged**